src/main/java/io/hologres/flink/example/HologresSinkExample.java

```diff
@@ -0,0 +1,74 @@
+package io.hologres.flink.example;
+
+import io.hologres.flink.ordergen.OrderTableSource;
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.CommandLineParser;
+import org.apache.commons.cli.DefaultParser;
+import org.apache.commons.cli.Options;
+import org.apache.flink.api.common.typeinfo.Types;
+import org.apache.flink.table.api.DataTypes;
+import org.apache.flink.table.api.EnvironmentSettings;
+import org.apache.flink.table.api.TableEnvironment;
+import org.apache.flink.table.api.TableSchema;
+
+public class HologresSinkExample {
```

Comment:
is this the jdbc or rpc sink? can we specify that in the class name?

Reply:
I think we should hide these implementation details